README.md

```diff
@@ -24,11 +24,11 @@ Using Gradle makes building WPILib very straightforward. It only has a few depen
 ## Requirements
 
 - A C++ compiler
-    - On Linux, gcc works fine
+    - On Linux, `gcc` works fine
     - On Windows, you need Visual Studio 2015 (the free community edition works fine).
       Make sure to select the C++ Programming Language for installation
 - [ARM Compiler Toolchain](http://first.wpi.edu/FRC/roborio/toolchains/)
-  * Note that for 2017-2018 and beyond, you will need version 5 or greater of gcc
+  * Note that for 2017-2018 and beyond, you will need version 5 or greater of `gcc`
```

Comment:
In this context, gcc is the name of the compiler not a piece of code.

Reply:
If, contextually, it's being used as the name of the compiler, then shouldn't it be GCC?  Keep in mind that `gcc` is also the command to invoke GCC.

Reply:
I would prefer GCC, as that’s the name of the entire suite.

Reply:
I've made the change and amended my commit.